src/main/java/com/matera/account/account/Account.java

```diff
@@ -28,7 +30,9 @@ public Account(UUID clientId, AccountDTO accountDTO) {
 
     private UUID clientId;
 
-    private Integer accountTypeId;
+    @ManyToOne
+    @JoinColumn(name = "account_type_id")
+    private AccountType accountTypeId;
```

Comment:
Since it changed to reference the full object, you can rename this attribut to just "accountType" :)